argparser.hpp

```diff
@@ -0,0 +1,17 @@
+#include <Windows.h>
+#include <tchar.h>
+#include "cpp_tstring.hpp"
+
+extern TCHAR helpMessage[];
+
+struct ArgumentParser {
```

Comment:
> It looks rather dangerous to have this many members exposed.

Do you mean that a junior developer might accidentally forget the second equal sign when comparing the value, so I should put getters instead?

Reply:
There can be a function for each, but there should also be a function that can query many properties using a string key. This can be two separate interfaces with multiple inheritance so it's cleaner.

Reply:
Added the getter functions in commit 580325cc6480d31f57a2bb5a50017e145eefaa82.
Made it be 2 interfaces in commit f6ab067d326cb54bef0e2874f28406117cdc2297. It made the code bigger, since C++ requires me to declare the functions inside the class as virtual either way.